active_directory/metadata.csv

```diff
@@ -22,15 +22,10 @@ active_directory.dra.outbound.objects.persec",gauge,,objects,second,"The number
 active_directory.dra.outbound.properties.persec",gauge,,properties,second,"The number of properties sent per second. This counter tells you whether a source server is returning objects or not. Sometimes, the server might stop working correctly and not return objects quickly or at all.",-1,active_directory,ad dra outbound properties
```

Comment:
not your fault but since we're here, can you fix line 21? There's an extra quote breaking the format